client/src/pages/Notification/EditChannelPopup/EditDiscordChannelPopup.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  EditDiscordChannelForm,
+  ManagmentServiceApiInstance,
+} from 'api/managment-service';
+import OverlayComponent from 'components/OverlayComponent/OverlayComponent';
+import { useState } from 'react';
+import { Form } from 'react-router-dom';
+import './EditChannelPopup.scss';
+import discordIcon from 'assets/discord-icon.png';
+import ActionButton, {
+  ActionButtonColor,
+} from 'components/ActionButton/ActionButton';
+import LabelInput, {
+  nonEmptyFieldValidation,
+} from 'components/LabelInput/LabelInput';
+import NewChannelPopupHeader from 'pages/Notification/NewChannelPopupHeader/NewChannelPopupHeader';
+import { EditChannelPopupProps } from './EditChannelPopup';
+import { discordWebhookValidation } from 'pages/Notification/NotificationTable/DiscordTable';
+
+interface EditDiscordChannelPopupProps extends EditChannelPopupProps {
+  id: string;
+  name: string;
+  webhookUrl: string;
+}
+
+const EditDiscordChannelPopup = ({
+  id,
+  name,
+  webhookUrl,
+  isDisplayed,
+  setIsDisplayed,
+  onSubmit,
+}: EditDiscordChannelPopupProps) => {
```

Comment:
Maybe there is a way to merge discord and slack popup into generic one? I see many commmon elements

Reply:
It was generic previously,and it caused few head aches. This way we can have more flexibility in methods, validators and details

Reply:
The stuff that is repeated is mosty "configuration" and not logic

Reply:
Okay, makes sense